src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/CollectionInstrumentSvcClient.java

```diff
@@ -14,8 +45,40 @@
    * @return list of collection instruments matching the search string
    * @throws RestClientException something went wrong making http call
    */
-  List<CollectionInstrumentDTO> requestCollectionInstruments(String searchString)
-      throws RestClientException;
+  @Retryable(
+      value = {RestClientException.class},
+      maxAttemptsExpression = "#{${retries.maxAttempts}}",
+      backoff = @Backoff(delayExpression = "#{${retries.backoff}}"))
+  public List<CollectionInstrumentDTO> requestCollectionInstruments(String searchString)
+      throws RestClientException {
+
+    MultiValueMap<String, String> queryParams = new LinkedMultiValueMap<>();
+    queryParams.add("searchString", searchString);
+
+    UriComponents uriComponents =
+        restUtility.createUriComponents(
+            appConfig.getCollectionInstrumentSvc().getRequestCollectionInstruments(), queryParams);
+
+    HttpEntity<List<CollectionInstrumentDTO>> httpEntity = restUtility.createHttpEntity(null);
+
+    ResponseEntity<String> responseEntity =
+        restTemplate.exchange(uriComponents.toUri(), HttpMethod.GET, httpEntity, String.class);
+
+    List<CollectionInstrumentDTO> result = null;
+    if (responseEntity != null && responseEntity.getStatusCode().is2xxSuccessful()) {
```

Comment:
This is a super ugly way of handling both the unmarshalling of the JSON and any errors. We should make it consistent with the other REST client code you've written